tuist/tuist
# Add tuist cloud project delete command

### Short description 📝

Adds a new command `tuist cloud project delete tuist --organization tuist`.

### How to test the changes locally 🧐

- Create a new project with `tuist cloud project create test`
- The project should be listed via `tuist cloud project list`
- Run `tuist cloud project delete test` (with an optional organization flag)
- Ensure the project is deleted – it shouldn't appear in the `tuist cloud project list` output

### Contributor checklist ✅


### Reviewer checklist ✅
